test/throw.test.js

```diff
@@ -17,12 +17,13 @@ test('Fastify should throw on wrong options', t => {
 test('Fastify should throw on multiple assignment to the same route', t => {
   t.plan(1)
   const fastify = Fastify()
-  fastify.get('/', () => {})
-  fastify.get('/', () => {})
 
-  fastify.ready(err => {
-    t.is(err.message, "Method 'GET' already declared for route '/' with constraints '{}'")
-  })
+  try {
+    fastify.get('/', () => {})
+    fastify.get('/', () => {})
```

Comment:
I'd add a t.fail bellow of this statement.

Reply:
Done in 0849c1b 👍